packages/cli/src/index.ts

```diff
@@ -7,21 +7,9 @@ import { bot, createBotDeprecate, deployBotDeprecate, saveBotDeprecate } from '.
 import { bulk } from './bulk';
 import { project } from './project';
 import { deleteObject, get, patch, post, put } from './rest';
-import { FileSystemStorage } from './storage';
-import { onUnauthenticated } from './util/client';
-
-export let medplum: MedplumClient;
-
-export async function main(medplumClient: MedplumClient, argv: string[]): Promise<void> {
-  medplum = medplumClient;
 
+export async function main(argv: string[]): Promise<void> {
   try {
-    const clientId = process.env['MEDPLUM_CLIENT_ID'];
-    const clientSecret = process.env['MEDPLUM_CLIENT_SECRET'];
-    if (clientId && clientSecret) {
-      medplumClient.setBasicAuth(clientId, clientSecret);
-      await medplumClient.startClientLogin(clientId, clientSecret);
-    }
```

Comment:
move to https://github.com/medplum/medplum/pull/2197/files#diff-80e8d5a02c540100cc38f9c1240d73742010ee4b2f195e615943ac89d67d5bc7